### PR TITLE
Intro page Dark mode 

### PR DESCRIPTION
**Pull request by Gtwatt1**
Added Dark mode to the intro page

### CODE REVIEW

File: RxMarbles/HelpViewController.swift [hunk 1]
```diff
@@ -551,7 +553,17 @@ class HelpViewController: AnimatedPagingScrollViewController {
     }
     
     private func _configureSharePage() {
-        let navBar = RxMarbles.Image.navBarShare.imageView()
+        var navBar = UIImageView()
+        if #available(iOS 13.0, *) {
+            if self.traitCollection.userInterfaceStyle == .dark {
```
Comment (yury) [inline RxMarbles/HelpViewController.swift]: @Gtwatt1 it is better to use assets appearance and not introduce new var (`navBarShareDark`)
image

File: RxMarbles/Image.swift [hunk 2]
```diff
@@ -45,6 +45,8 @@ struct Image {
     static let ellipse2     = _named("anjlab_ellipse2")
     
     static let navBarShare          = _named("navBarShare")
+    static let navBarShareDark          = _named("navBarShareDark")
```
Comment (yury) [inline RxMarbles/Image.swift]: No need for extra var

**Review by yury — CHANGES_REQUESTED** [review]
Also remove changes from Xcode project file.

File: RxMarbles/HelpViewController.swift (hunk 1 above)
Comment (Gtwatt1) [reply RxMarbles/HelpViewController.swift]: thanks and it has been fixed.

File: RxMarbles/Image.swift (hunk 2 above)
Comment (Gtwatt1) [reply RxMarbles/Image.swift]: fixed

**Comment by yury** [comment]
Thank you @Gtwatt1

**Pull request merged**